### PR TITLE
Bugfix/440214 fix sorting

### PR DESCRIPTION
**Pull request by aldeez988**
## Checklist

- [ ] Content has no typos
- [ ] Correct inline validation errors
- [ ] Back link and next page navigation
- [ ] Removed all unnecessary console logs
- [ ] Removed all commented out code
- [ ] Updated README / confluence

### CODE REVIEW

File: app/routes/models/application-list.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const { getStyleClassByStatus } = require('../../constants/status')
 const keys = require('../../session/keys')
 const { serviceUri, endemics } = require('../../config')
 const { upperFirstLetter } = require('../../lib/display-helper')
+const { sort } = require('../agreement')
```
Comment (alex742) [inline app/routes/models/application-list.js:8]: Imported but not used?

File: app/routes/models/application-list.js [hunk 2]
```diff
@@ -13,6 +14,7 @@ const viewModel = (request, page) => {
 }
 
 const getApplicationTableHeader = (sortField) => {
+  console.log('>>>>>>>>>>>>>>>>', sortField)
```
Comment (alex742) [inline app/routes/models/application-list.js:17]: Can we remove this

File: app/routes/models/application-list.js (hunk 1 above)
Comment (aldeez988) [reply app/routes/models/application-list.js]: removed

File: app/routes/models/application-list.js (hunk 2 above)
Comment (aldeez988) [reply app/routes/models/application-list.js]: removed

**Review by alex742 — APPROVED** [review]

**Pull request merged**